src/Tizen.Multimedia.Remoting/MediaController/MediaController.cs

```diff
@@ -322,7 +274,56 @@ public MediaControlRepeatMode GetRepeatMode()
         }
 
         /// <summary>
-        /// Sends playback command to the server.</summary>
+        /// Requests command to the server.
+        /// </summary>
+        /// <remarks>
+        /// The client can request the server to execute <see cref="PlaybackCommand"/> or <see cref="ShuffleModeCommand"/> or
+        /// <see cref="RepeatModeCommand"/> or <see cref="CustomCommand"/>, <br/>
+        /// and then, the client receive the result of each request(command).
+        /// </remarks>
+        /// <param name="command">A <see cref="Command"/> class.</param>
+        /// <exception cref="InvalidOperationException">
+        ///     The server has already been stopped.<br/>
+        ///     -or-<br/>
+        ///     An internal error occurs.
+        /// </exception>
+        /// <exception cref="ObjectDisposedException">The <see cref="MediaControllerManager"/> has already been disposed of.</exception>
+        /// <returns>The request ID for <paramref name="command"/>.</returns>
```

Comment:
Description about returns is wrong

What is returing `int` value?
If it just means success it should be change to bool 

and in C# API, does not use error code with return value `int`
if you want to inform the error, do use `Exception` instead of error code

Reply:
```<returns>The request ID for``` is typo. It did intend result code.
Because, I thought it's just not error but information including status , ... . 
But I check it again with native fw owner, it's error code.
So I'll modify return type as ```Task``` and if there's error I'll throw exception.